### ISSUE TITLE
character encoding problem

### ISSUE THREAD

**Issue by Bescu** [rburhum/arcgis-ogr#2]
Ok thank you very much for this plugin!!

I have a little problem with the character encoding : the "spécials character" in my spatialite database are not read good in Arcgis. For exemple : 

When I add my spatialite layer in Qgis with the UTF-8 encoding I have : "propriétaire" and if i add my layer with the plugin in ArcMap I have : "propriÃ©taire".

I think that the plugin automaticly choose the Latin-1 encoding (or something like that) because when I add my layer in Qgis with the latin-1 encoding it's the same thing.

I think that the problem is we can't choose the encoding when we add a layer.

**Comment by rburhum** [comment]
That sounds right. The Plugin itself doesn't attempt to do any encoding translation. Perhaps adding a setting to choose the encoding. Again, could I get a sample dataset. And thanks for testing Bescu! I assume the installer worked fine for you?

**Comment by Bescu** [comment]
Yes the installer worked fine for me!! Very nice! Sorry for my late answer. You can find a sample dataset with a spatialite DB with the encoding "problem" and a MapInfo TAB that I can't open with the plugin : http://dl.free.fr/nLL0GYybn
Thanks a lot for this very interesting plugin!

**Comment by rburhum** [comment]
Finally got around to looking into this issue yesterday and today. I tracked it down to be a problem with the implementation of the GDAL/Swig C# bindings. I submitted a bug there with the description of the problem http://trac.osgeo.org/gdal/ticket/4971

In the meantime, I committed a workaround that will make this work. https://github.com/RBURHUM/arcgis-ogr/commit/341a701b0c1a096f4da304d7965403b7618d55e9
